src/Ng/CommandHelpers.cs

```diff
@@ -119,7 +119,8 @@ public static StorageFactory CreateStorageFactory(IDictionary<string, string> ar
                 { Arguments.StorageKeyValue, Arguments.StorageKeyValue },
                 { Arguments.StorageContainer, Arguments.StorageContainer },
                 { Arguments.StoragePath, Arguments.StoragePath },
-                { Arguments.StorageSuffix, Arguments.StorageSuffix }
+                { Arguments.StorageSuffix, Arguments.StorageSuffix },
+                { Arguments.StorageOperationMaxExecutionTimeInSeconds, Arguments.StorageOperationMaxExecutionTimeInSeconds },
```

Comment:
Added a default, changed the int? to int  but kept the configuration of the maxtimeout. I think that having the be able to configure different timeouts for different environments can be beneficial.